### PR TITLE
39-Life@TCET Home Page

### PR DESCRIPTION
**Pull request by Alstudd**
Created components and pages for Life@TCET Home page

### CODE REVIEW

File: src/styles/life-at-tcet-home.css [hunk 1]
```diff
@@ -0,0 +1,129 @@
+@import url('https://fonts.googleapis.com/css2?family=Fraunces&display=swap');
+
+
+.main-section {
```
Comment (JayeshVP24) [inline src/styles/life-at-tcet-home.css:4]: never give fix width and heights

Comment (JayeshVP24) [reply src/styles/life-at-tcet-home.css:4]: take your time, move to tailwindCSS

**Review by JayeshVP24 — CHANGES_REQUESTED** [review]

File: src/pages/life-at-tcet/index.astro [hunk 2]
```diff
@@ -0,0 +1,22 @@
+---
+import Layout from '../../layouts/Layout.astro'
+import FirstCards from "../../components/LifeAtTCET/Home/FirstCards.astro";
+import SecondCards from "../../components/LifeAtTCET/Home/SecondCards.astro";
+import ThirdCards from "../../components/LifeAtTCET/Home/ThirdCards.astro";
+---
+<Layout title="Life@TCET" description='Life@TCET-Home page'>
+    <div class="sm:items-center max-w-[831px] flex flex-col gap-[15px] pl-3 rounded-[7px]">
+        <!-- Fraunces font-family using font-title -->
+        <h1 class="font-title sm:text-center text-4xl font-normal leading-[44px] tracking-[-0.02em] text-left text-[#101828">Life@TCET</h1>
```
Comment (JayeshVP24) [inline src/pages/life-at-tcet/index.astro:10]: u forgot the bracked at text color

File: src/pages/life-at-tcet/index.astro [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+        <FirstCards />
+        <SecondCards />
+        <h2 class="text-2xl font-semibold leading-8 tracking-normal text-left">Life @ TCET</h2>
+        <ul class="pl-[30px]">
```
Comment (JayeshVP24) [inline src/pages/life-at-tcet/index.astro:14]: remove pixel and use tailwind unit
pl-6

File: src/pages/life-at-tcet/index.astro [hunk 4]
```diff
@@ -0,0 +1,22 @@
...
+            <li style="list-style-type: disc;" class="sm:w-[320px] text-base font-medium leading-6 tracking-normal text-left text-[#757575] w-[750px]">TCET Acheivements:A Mark of Quality Education</li>
```
Comment (JayeshVP24) [inline src/pages/life-at-tcet/index.astro:15]: remove w-[320px]
never use fixed widths unless absolutely necessary such as with images
remove w-[750px]

File: src/pages/life-at-tcet/index.astro [hunk 5]
```diff
@@ -0,0 +1,22 @@
...
+            <li style="list-style-type: disc;" class="sm:w-[320px] text-base font-medium leading-6 tracking-normal text-left text-[#757575] w-[750px]">Institute is NAAC accredited with grade “A” in 1 Cycle for 5 years w.e.f. 30/10/2017. Undergraduate Programs of E&TC IT, COMP, and ELEX are accredited by National Board of Accreditation, New Delhi w.e.f. 1.7.2019 for 3 years which is extended upto 30.6.2022 in the current academic year</li>
```
Comment (JayeshVP24) [inline src/pages/life-at-tcet/index.astro:16]: remove width

File: src/pages/life-at-tcet/index.astro [hunk 6]
```diff
@@ -0,0 +1,22 @@
...
+            <li style="list-style-type: disc;" class="sm:w-[320px] text-base font-medium leading-6 tracking-normal text-left text-[#757575] w-[750px]">Institute is ranked among top 200 colleges in the country where the institute is ranked 193 in NIRF India Ranking 2019 in the Engineering category and 68th & 78th in All India Rank by Outlook survey published in June 2019 & May 2018 respectively</li>
```
Comment (JayeshVP24) [inline src/pages/life-at-tcet/index.astro:17]: remove width

**Review by JayeshVP24 — CHANGES_REQUESTED** [review]

File: src/pages/life-at-tcet/index.astro [hunk 7]
```diff
@@ -0,0 +1,22 @@
+---
+import Layout from '../../layouts/Layout.astro'
+import FirstCards from "../../components/LifeAtTCET/Home/FirstCards.astro";
+import SecondCards from "../../components/LifeAtTCET/Home/SecondCards.astro";
+import ThirdCards from "../../components/LifeAtTCET/Home/ThirdCards.astro";
+---
+<Layout title="Life@TCET" description='Life@TCET-Home page'>
+    <div class="sm:items-center flex flex-col gap-4 pl-3 rounded-lg">
```
Comment (JayeshVP24) [inline src/pages/life-at-tcet/index.astro:8]: remove pl-3 and add `p-4 lg:p-16`

File: src/pages/life-at-tcet/index.astro [hunk 8]
```diff
@@ -0,0 +1,22 @@
...
+        <ul class="pl-6">
+            <li style="list-style-type: disc;" class="text-base font-medium leading-6 tracking-normal text-left text-[#757575]">TCET Acheivements:A Mark of Quality Education</li>
+            <li style="list-style-type: disc;" class="text-base font-medium leading-6 tracking-normal text-left text-[#757575]">Institute is NAAC accredited with grade “A” in 1 Cycle for 5 years w.e.f. 30/10/2017. Undergraduate Programs of E&TC IT, COMP, and ELEX are accredited by National Board of Accreditation, New Delhi w.e.f. 1.7.2019 for 3 years which is extended upto 30.6.2022 in the current academic year</li>
+            <li style="list-style-type: disc;" class="text-base font-medium leading-6 tracking-normal text-left text-[#757575]">Institute is ranked among top 200 colleges in the country where the institute is ranked 193 in NIRF India Ranking 2019 in the Engineering category and 68th & 78th in All India Rank by Outlook survey published in June 2019 & May 2018 respectively</li>
+        </ul>
+        <h2 class="text-2xl font-semibold leading-8 tracking-normal text-left">Facilities@TCET</h2>
+        <ThirdCards />
```
Comment (JayeshVP24) [inline src/pages/life-at-tcet/index.astro:20]: text-left is default ig, u can remove it

File: src/pages/life-at-tcet/index.astro [hunk 9]
```diff
@@ -0,0 +1,22 @@
...
+import FirstCards from "../../components/LifeAtTCET/Home/FirstCards.astro";
+import SecondCards from "../../components/LifeAtTCET/Home/SecondCards.astro";
+import ThirdCards from "../../components/LifeAtTCET/Home/ThirdCards.astro";
+---
+<Layout title="Life@TCET" description='Life@TCET-Home page'>
+    <div class="sm:items-center flex flex-col gap-4 pl-3 rounded-lg">
+        <!-- Fraunces font-family using font-title -->
+        <h1 class="font-title sm:text-center text-4xl font-normal leading-8 tracking-[-0.02em] text-left text-[#101828]">Life@TCET</h1>
+        <FirstCards />
+        <SecondCards />
```
Comment (JayeshVP24) [inline src/pages/life-at-tcet/index.astro:12]: wrap in one <section>

File: src/pages/life-at-tcet/index.astro [hunk 10]
```diff
@@ -0,0 +1,22 @@
...
+        <h2 class="text-2xl font-semibold leading-8 tracking-normal text-left">Life @ TCET</h2>
+        <ul class="pl-6">
+            <li style="list-style-type: disc;" class="text-base font-medium leading-6 tracking-normal text-left text-[#757575]">TCET Acheivements:A Mark of Quality Education</li>
+            <li style="list-style-type: disc;" class="text-base font-medium leading-6 tracking-normal text-left text-[#757575]">Institute is NAAC accredited with grade “A” in 1 Cycle for 5 years w.e.f. 30/10/2017. Undergraduate Programs of E&TC IT, COMP, and ELEX are accredited by National Board of Accreditation, New Delhi w.e.f. 1.7.2019 for 3 years which is extended upto 30.6.2022 in the current academic year</li>
+            <li style="list-style-type: disc;" class="text-base font-medium leading-6 tracking-normal text-left text-[#757575]">Institute is ranked among top 200 colleges in the country where the institute is ranked 193 in NIRF India Ranking 2019 in the Engineering category and 68th & 78th in All India Rank by Outlook survey published in June 2019 & May 2018 respectively</li>
+        </ul>
```
Comment (JayeshVP24) [inline src/pages/life-at-tcet/index.astro:18]: wrap in one <section>

File: src/pages/life-at-tcet/index.astro (hunk 8 above)
Comment (JayeshVP24) [inline src/pages/life-at-tcet/index.astro:20]: wrap in one <section>

**Review by JayeshVP24 — CHANGES_REQUESTED** [review]

**Review by kamalika0363 — APPROVED** [review]
Approved Changes

**Pull request merged**